assets/stylesheet.css

```diff
@@ -210,12 +241,25 @@ a:visited {
     color: #FFFFFF;
 }
 
+#update {
```

Comment:
suggestion: It seems like most of this block is duplicating rules for other buttons defined just above. I wonder if instead of using `id`s (which also require you to list each one), we could use a couple of classes instead? Like maybe the rules here might be declared as a class called `.button` or something along those lines, and then the couple of rules that the other buttons need in addition to these might be called `.button-wide` or something. So then the update button would be defined with `<button id="update" class="button">`, and then a submit button would be something like `<input type="submit" value="Submit" id="submit" class="button button-wide"/>`.